src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -1194,11 +1337,17 @@ await DownloadFileAsync(file.FilePath, destination, cancellationToken)
             float latitude,
             float longitude,
             InlineKeyboardMarkup replyMarkup = default,
-            CancellationToken cancellationToken = default
+            CancellationToken cancellationToken = default,
+            float horizontalAccuracy = default,
+            int heading = default,
+            int proximityAlertRadius = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.